main.go

```diff
@@ -250,12 +250,14 @@ func Flash(pkgName, port string, options *compileopts.Options) error {
 
 			err := touchSerialPortAt1200bps(port)
 			if err != nil {
-				return &commandError{"failed to reset port", result.Binary, err}
+				return &commandError{"failed to reset port", "", err}
```

Comment:
At the time of processing this part, result.Binary is undefined, so it is assumed to be empty.